### ISSUE TITLE
[Issue]: impish Release does not have a Release file

### ISSUE THREAD

**Issue by chrisolsen** [jellyfin/jellyfin#7562]
### Please describe your bug

When running the following I get an error message of `The repository 'https://repo.jellyfin.org/debian impish Release' does not have a Release file.`

```
sudo apt install ca-certificates
sudo apt install apt-transport-https
curl https://repo.jellyfin.org/debian/jellyfin_team.gpg.key | gpg --dearmor | sudo tee /usr/share/keyrings/jellyfin-archive-keyring.gpg >/dev/null
echo "deb [signed-by=/usr/share/keyrings/jellyfin-archive-keyring.gpg arch=$( dpkg --print-architecture )] https://repo.jellyfin.org/debian $( lsb_release -c -s ) main" | sudo tee /etc/apt/sources.list.d/jellyfin.list
sudo apt update
```

### Jellyfin Version

10.8.0

### Environment

```markdown
- OS: Pop!_OS 21.10 impish
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Shadowghost** [comment]
Impish will only be available when 10.8 is released. The beta is already available as manual download for impish.

**Comment by chrisolsen** [comment]
That worked :+1:
